Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -324,7 +324,11 @@ void UpdateLineBreakMode()
 		void UpdateMaxLines()
 		{
 			SetSingleLine(Element.MaxLines == 1);
-			SetMaxLines(Element.MaxLines > 0 ? Element.MaxLines : 1);
+
+			if (Element.MaxLines == (int)Label.MaxLinesProperty.DefaultValue)
```

Comment:
This will also be a breaking change because before it was always 1

Reply:
Before it was always MaxValue on Android until UpdateMaxLines() was introduced and set the default to 1.

Reply:
> Before it was always MaxValue on Android until UpdateMaxLines() was introduced and set the default to 1.

It was always 1 or 100, depending on what the LineBreakMode setting was. (Though it probably _should_ have been MaxValue rather than an arbitrary 100).

Reply:
@hartez Are you talking specifically of Android? Where is it set to 100 there? I didn't see it in `LabelRenderer`.

Also, I just created a brand new app with 3.3.0-pre1 and added a long label with `LineBreakMode="WordWrap"`, but all I see is a single line and no wrapping. I'd expect this to work without having to manually set any MaxLines. Was this always the existing behavior or did MaxLines() (which was checked in on 8/16) break this?

Reply:
The new MaxLines stuff broke it.

Reply:
> Are you talking specifically of Android? Where is it set to 100 there? I didn't see it in LabelRenderer.

Check out `TextViewExtensions`.